extensions/couchbase/controllerservices/CouchbaseClusterService.cpp

```diff
@@ -70,16 +71,57 @@ nonstd::expected<CouchbaseUpsertResult, CouchbaseErrorType> CouchbaseClient::ups
       document_id, collection.bucket_name, collection.scope_name, collection.collection_name, upsert_err.ec(), upsert_err.message());
     return nonstd::make_unexpected(CouchbaseErrorType::FATAL);
   } else {
-    const uint64_t partition_uuid = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_uuid() : 0);
-    const uint64_t sequence_number = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->sequence_number() : 0);
-    const uint16_t partition_id = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_id() : 0);
-    return CouchbaseUpsertResult {
-      collection.bucket_name,
-      upsert_resp.cas().value(),
-      partition_uuid,
-      sequence_number,
-      partition_id
-    };
+    CouchbaseUpsertResult result;
+    result.bucket_name = collection.bucket_name;
+    result.cas = upsert_resp.cas().value();
+    result.partition_uuid = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_uuid() : 0);
+    result.sequence_number = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->sequence_number() : 0);
+    result.partition_id = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_id() : 0);
+    return result;
```

Comment:
Unfortunately because of the added inheritance in the CouchbaseUpsertResult, we could no longer use the previous format, but I changed it to list initialization in https://github.com/apache/nifi-minifi-cpp/pull/1881/commits/6b8d35c03e88a5d1e10bfdeab869a17dbe51a71a